src/main/java/org/phoebus/channelfinder/processors/AAChannelProcessor.java

```diff
@@ -108,10 +108,18 @@ public long process(List<Channel> channels) throws JsonProcessingException {
                     .filter(xmlProperty -> archivePropertyName.equalsIgnoreCase(xmlProperty.getName()))
                     .findFirst();
             if (archiveProperty.isPresent()) {
-                try {
-                    addChannelChange(channel, aaArchivePVS, policyLists, archiveProperty);
-                } catch (Exception e) {
-                    logger.log(Level.WARNING, String.format("Failed to process %s", channel), e);
+                List<String> archivers = Optional.of(channel.getProperties().stream()
+                        .filter(xmlProperty -> archiverPropertyName.equalsIgnoreCase(xmlProperty.getName()))
+                        .map(Property::getValue)
+                        .collect(Collectors.toList()))
+                        .filter(list -> !list.isEmpty())
+                        .orElse(defaultArchiver);
+                for(String archiverAlias : archivers) {
+                    try {
+                        addChannelChange(channel, aaArchivePVS, policyLists, archiveProperty, archiverAlias);
```

Comment:
This can be done inside the stream with a .forEach, so then there should be no need to do a collect.

Reply:
okay I updated this and now you can also specify a comma separated list for the `archiver` info tag